features/manage-position/Create.tsx

```diff
@@ -96,8 +96,10 @@ const Create = () => {
     // is intended to encourage users to maximize their capital efficiency.
     const maxTokensToCreate = _gcr > 0 ? collateral / _gcr : 0;
     // Unlike the min collateral, we're ok if we round down the tokens slightly as round down
-    // can only increase the position's CR and maintain it above the GCR constraint
-    setTokens(maxTokensToCreate.toFixed(4));
+    // can only increase the position's CR and maintain it above the GCR constraint. However, we'll
+    // subtract a tiny amount of tokens because the prevision division might unintentionally floor
+    // the quotient.
+    setTokens((maxTokensToCreate - 1).toFixed(4));
```

Comment:
Why do you have to subtract 1? Can’t you just subtract like 0.00005 or something?

Reply:
the current check for enabling the Create button is:

`transactionCR > GCR` where `transactionCR = collateral / tokens` and `collateral` and `tokens` are the string values (not numbers) set by these two `setStates` from Min and Max functions. So this is by far the easiest way to do it without changing other logic (which we probably should but I wanted to keep out of scope).

This admittedly won't scale for certain types of token where 1 synthetic represents a lot of real value.

Reply:
In the interest of speed, I think this is good for now!

I don't quite understand why we would subtract from the quotient if we're worried the division may have floored it. I would imagine that we'd want to _add_ in that case. I also don't see why floating point division would ever ceil or floor unless it lost precision. In the case of lost precision, I'm actually not too sure whether we have _any_ guarantees about the error without knowing how large the numbers are.